requests_mock/adapter.py

```diff
@@ -245,6 +249,7 @@ def send(self, request, **kwargs):
             if resp is not None:
                 request._matcher = weakref.ref(matcher)
                 resp.connection = self
+                logger.debug('{} \"{} \" {}'.format(request._request.url,request._request.method, resp.status_code))
```

Comment:
Where did the format come from? The space after method seems weird. It prints like: 

`DEBUG:requests_mock.adapter:http://test.com/ "GET " 200`

I would think it would either follow apache format, or maybe `GET http://test.com 200`.